app/main.py

```diff
@@ -2,0 +1,9 @@
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+    if command_parts[0] == "cp":
+        raise ValueError("Bad command")
```

Comment:
This line should be executed if the command does not start with 'cp'. Adjust the condition in the previous line accordingly.